bpfd-operator/controllers/bpfd-agent/internal/bpfd-core.go

```diff
@@ -246,21 +223,25 @@ func GetMapsForID(id uint32) (map[string]string, error) {
 
 // Convert a list result into a set of kernel info annotations
 func Build_kernel_info_annotations(p *gobpfd.ListResponse_ListResult) map[string]string {
```

Comment:
I didn't see this anywhere, but can you also now make sure to update the BpfPrograms for OUR programs (programs loaded by bpfd) with these annotations as well? 

(Doing this in a separate PR is alright)

Reply:
#744